src/components/IndexPage/testimonial.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Body from "styles/Body"
+import { HeaderLineBelow } from "styles/Header"
+import Section from "styles/Section"
+import styled from "styled-components"
+
+const SectionPad = styled(Section)`
+  padding-top: 0px;
+`
+
+const Content = styled.div`
+  max-width: 600px;
+  margin: 0px auto;
+`
+
+const AuthorInfo = styled(Body)`
+  text-align: right;
+  font-style: italic;
+`
+
+const Testimonial = ({ quote, author, role }) => (
+  <SectionPad>
+    <Content>
+      <Body dangerouslySetInnerHTML={{ __html: quote }} />
+      <AuthorInfo>{author}</AuthorInfo>
+      <AuthorInfo>{role}</AuthorInfo>
+      <br />
+      <Body>
+        Think you could take advantage of computation in your research? Please
```

Comment:
Think you could take advantage of computation in your research?
<Button>Work with us</Button>

Is more of a call to action and I think would get more responses. If you wanna leave it as is, there's a missing period after the second sentence.